TicketsDemo.Data/DTO/PriceParametersDTO.cs

```diff
@@ -0,0 +1,10 @@
+﻿using TicketsDemo.Data.Entities;
+
+namespace TicketsDemo.Data.DTO
+{
+    public class PriceParametersDTO
+    {
+        public PlaceInRun PlaceInRun { get; set; }
+        public Ticket Ticket { get; set; }
```

Comment:
Price calcualation happens based on:
1. Place in run
2. Day, for which ticket is to be bought
... any other paratemeters to be added in future ...